x-pack/plugins/session_view/public/components/detail_panel_alert_tab/index.tsx

```diff
@@ -118,17 +118,19 @@ export const DetailPanelAlertTab = ({
       )}
 
       {viewMode === VIEW_MODE_LIST
-        ? filteredAlerts.map((event) => {
+        ? alerts.map((event) => {
             const key = event.kibana?.alert?.uuid;
```

Comment:
do we need a fallback in case we have multiples `undefined` keys? i.e 

```
// ...
import uuid from 'uuid';
// ...
const key = event.kibana?.alert?.uuid || uuid.v4();
```

or

```
alerts.map((event, index) => {
    const key = event.kibana?.alert?.uuid || index;
    //...
})
```